pkg/controller/chaosengine/chaosengine_controller.go

```diff
@@ -596,3 +605,15 @@ func setChaosResourceImage() {
 		engine.Instance.Spec.Components.Runner.Image = ChaosRunnerImage
 	}
 }
+
+func getChaosType() error {
+
+	if engine.Instance.Spec.ChaosType == "" {
+		engine.Instance.Spec.ChaosType = chaosTypes.DefaultChaosType
+
+	}
+	if engine.Instance.Spec.ChaosType != "app" && engine.Instance.Spec.ChaosType != "infra" {
```

Comment:
Handle the case if infra, with something like no supported at the moment.
cc: @ksatchit @chandankumar4

Reply:
@rahul, I don't get it. Can you please explain?